index.html

```diff
@@ -268,8 +269,16 @@ <h2 class="work-title">My Recent Works</h2>
     <!-- about section -->
     <section class="about-section">
       <div class="about-text">
-        <img src="icons/desktopimage/illustration-about me-1-desktop@2x.png" alt="" class="image1">
-        <img src="icons/desktopimage/illustration-about-me-2-desktop@2x.png" alt="" class="image2">
+        <img
+          src="icons/desktopimage/illustration-about me-1-desktop@2x.png"
+          alt=""
+          class="image1"
+        />
+        <img
+          src="icons/desktopimage/illustration-about-me-2-desktop@2x.png"
+          alt=""
+          class="image2"
+        />
 
         <h1 class="primary-text">About me</h1>
         <p class="secondary-text .bg-top">
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/193875178-cf0aca96-a66f-4590-87b2-e78f63580034.png)

- [ ] Your See Project buttons do not have the color contrast of 4.5:1. Kindly make this contrast due to accessibility purpose :blush: